src/conventional/sonde_tac2ioda.py

```diff
@@ -724,10 +728,12 @@ def getDayHour(str):
     :param str: The string to parse
     :return: (day, hour)
     """
+    global scale_wspd
     try:
         day = int(str[:2]) - 50
```

Comment:
may be good to leave a comment that WMO codes for TAC TEMP messages notes that day of month incremented by 50 indicates winds in knots

Reply:
I documented in the comments of the PR, but documenting every little thing in the code directly would be a nuisance. So decide if this is a show-stopper or just approve as the list of things to get to match the WMO manual on codes would be purely crazy.